CHANGELOG.md

```diff
@@ -2777,6 +2782,17 @@ Additions to existing modules
   pattern divides-refl q = divides q refl
   ```
 
+* Added  a new instance and new definitions to `Data.Nat.Divisibility`:
+  ```agda
+  instance quotient≢0 : m ∣ n → .{{NonZero n}} → NonZero quotient
+
+  equalityᵒ    : m ∣ n → n ≡ m * quotient
+  quotient-∣   : m ∣ n → quotient ∣ n
+  quotient>1   : m ∣ n → m < n → 1 < quotient
```

Comment:
After #2181 suggest that this be redefined in terms of `NonTrivial` defined there.